### PR TITLE
add transition between pages

### PR DESCRIPTION
**Pull request by Corentin7301**
Adding opacity transition between pages of the website.

### CODE REVIEW

**Comment by xlanex6** [comment]
@Corentin7301 

Thanks for your work 
is their a way to visualize the result before merging ? 
any  impact on performance ?

**Comment by Corentin7301** [comment]
@xlanex6 

No impact on performance, it's a Nuxt fonctionnality (https://nuxt.com/docs/getting-started/transitions).
Can't you see the result on the Netlify PR's build when you change the page?

File: assets/main.css [hunk 1]
```diff
@@ -99,3 +99,12 @@ h2 {
     @apply relative w-2 h-2 transition-all border-none rounded-full appearance-none bg-yellowDs;
   }
 }
+
+.page-enter-active,
+.page-leave-active {
+  transition: all 0.2s;
```
Comment (ipatate) [inline assets/main.css:105]: use tailwind please. @apply...

File: assets/main.css [hunk 2]
```diff
@@ -99,3 +99,12 @@ h2 {
...
+}
+.page-enter-from,
+.page-leave-to {
+  opacity: 0;
```
Comment (ipatate) [inline assets/main.css:109]: same.

File: layouts/default.vue [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="min-h-screen bg-gradient-to-b from-purple-50 to-purple-100">
+  <div class="min-h-screen bg-gradient-to-b from-darkPurple to-purpleDs">
```
Comment (ipatate) [inline layouts/default.vue:2]: the bg color isn't good for some pages
Screenshot 2022-12-08 at 14 28 50

**Review by ipatate — CHANGES_REQUESTED** [review]
Good idea 👍

**Comment by Corentin7301** [comment]
Normally all is good. I've changed vanilla css to Tailwind style and I've updated bg color in all pages (can you test all the pages please?).

**Comment by ipatate** [comment]
yeah, i'll look, thanks

**Comment by ipatate** [comment]
Désolé pas eu le temps de regarder encore.

**Pull request closed without merging**